baselines/teams.md

```diff
@@ -18,94 +22,93 @@ The **License Requirements** sections of this document assume the organization i
-    authenticated through the agency’s tenant, including unmanaged
+5.  **Anonymous users**: Teams users joining calls who are not
+    authenticated through the agency’s tenant; these users include unmanaged
     users, external users (except for B2B guests), and true anonymous
-    users, meaning users that are not logged in to any Microsoft or
-    organization account, such as dial-in users.[^1]
+    users (i.e., users who are not logged in to any Microsoft or
+    organization account, such as dial-in users[^1]).
 
 # Baseline Policies
 
 ## 1. Meeting Policies
 
-This section helps reduce security risks posed by the external participants during meetings. In this instance, the term “external participants” includes external users, Business to Business (B2B) guest users, unmanaged users, and anonymous users.
+This section helps reduce security risks posed by the external participants during meetings. In this instance, the term “external participants” includes external users, B2B guest users, unmanaged users, and anonymous users.
 
-This section helps reduce security risks related to the user permissions for recording Teams meetings and events. These policies and user permissions apply to meetings hosted by a user, during one-on-one calls, and group calls started by a user. Agencies should comply with any other applicable policies or legislation in addition to this guidance.
+This section helps reduce security risks related to the user permissions for recording Teams meetings and events. These policies and user permissions apply to meetings hosted by a user, as well as during one-on-one calls and group calls started by a user. Agencies should comply with any other applicable policies or legislation in addition to this guidance.
 
 ### Policies
 
 #### MS.TEAMS.1.1v1
 External meeting participants SHOULD NOT be enabled to request control of shared desktops or windows.
 
 <!--Policy: MS.TEAMS.1.1v1; Criticality: SHOULD -->
-- _Rationale:_ There is risk in granting an external participant control of a shared screen because the participant could potentially perform unauthorized actions on the shared screen. This policy reduces that risk by removing an external participant's ability to request control. However, if an agency has a legitimate use case to grant this control, it may be done on a case-by-case basis.
+- _Rationale:_ An external participant with control of a shared screen could potentially perform unauthorized actions on the shared screen. This policy reduces that risk by removing an external participant's ability to request control. However, if an agency has a legitimate use case to grant this control, it may be done on a case-by-case basis.
 - _Last modified:_ July 2023
-- _Note:_ This policy applies to the Global (Org-wide default) meeting policy, and custom meeting policies if they exist.
+- _Note:_ This policy applies to the Global (org-wide default) meeting policy, as well as custom meeting policies.
```

Comment:
I recommend keeping "Org-wide" capitalized, as that's the spelling Microsoft uses.

![image](https://github.com/cisagov/ScubaGear/assets/106177711/6cf6aadb-4189-424d-b1e6-750b7c51c1cc)

```suggestion
- _Note:_ This policy applies to the Global (Org-wide default) meeting policy, as well as custom meeting policies.
```